helm/charts/kratos/templates/deployment-kratos.yaml

```diff
@@ -85,6 +85,14 @@ spec:
           {{- if $migrationExtraEnv }}
             {{- toYaml $migrationExtraEnv | nindent 12 }}
           {{- end }}
+          {{- if .Values.kratos.automigration.resources }}
```

Comment:
This can be simplified using with instead of checking the if clause

Reply:
Done with 2aaa1aafef71fe31b88dac9a84cbe6af602edb89